Hackathon Prep Guide (Accelerated Developer Journey) (#2292)
dfinity/portal: docs/tutorials/hackathon-prep-course/7-obtaining-cycles.md

Author: Added a link to the previous tutorial chapter.

To obtain cycles, the Internet Computer Protocol's utility token, ICP, can be converted into cycles and transferred into a canister to be used to pay for that canister's consumed resources. Cycles have a fixed price in XDR in order to make canister costs predictable and independent of the price of ICP. One trillion cycles always correspond to one XDR.

> Since cycles are not a currency and are only used to pay for a canister's consumed resources, developers manage the distribution of cycles through a special canister called a cycles wallet. When a canister needs to use cycles, the canister's operations are executed using the cycles wallet's principal. 

Author: Revised to fix wording

dfx ledger --network ic balance
```

> After you have your ICP tokens ready, you will need to create a cycles wallet if you haven't already. To check and see if your identity already has a cycles wallet on the mainnet, you can use the following command to retrieve the current wallet addresses for your identity:

Reviewer: ```suggestion
After you have your ICP tokens ready, you will need to create a cycles wallet if you haven't already. To check and see if your identity already has a cycles wallet on the mainnet, you can use the following command:
```

```

> dfx wallet --network ic addresses

Reviewer: ```suggestion
dfx identity --ic get-wallet
```

```

> If your identity has a cycles wallet already, it will be returned in the output:
> 
> ```
> Id: lsa7b-dlc2s-vmdhm-skjw6-gv2p7-v7kjz-yppyo-mvwmg-ju6fc-k2kpm-dqe, Kind: Unknown, Role: Controller, Name: No name set.
> ```

Reviewer: ```suggestion
If your identity has a cycles wallet already, it will be returned in the output, otherwise you'll see an error.
```

> dfx ledger top-up 5o6tz-saaaa-aaaaa-qaacq-cai --network ic --amount 1.005

Reviewer: ```suggestion
dfx ledger top-up <CYCLE_WALLET_CANISTER_ID> --network ic --amount 1.005
```

Reviewer: we should add a previous step explaining how to get the canister id
```
dfx identity --ic get-wallet
```

Reviewer: where is the `vite-motoko-react` project coming from that we are deploying here? i think its helpful to link to it again, even if its used in previous sections of the guide
